MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## How to become a maintainer
+
+### 1. Be an active contributor
+
+An active contributor is someone who contributed regularly in the last 2 months
+and is one of the top 20 of contributors for the last 2 months.
+
+For example in the context of merged commit contributions, from September to
+November, the 20th developer has 20 commits.
+
+#### Contribution Types
+
+Contributions are not limited to code. Other contributions include PR reviews,
+discussion participations, test ownership, SIG leadership, etc. At the moment we
+are only able to measure PR contributions. As we improve our project tooling,
+we'll be able to measure and analyze other types of contributions and take those
+into account when evaluating a maintainer application.
+
+### 2. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+### 3. Find 3 senior maintainer sponsors
+
+As you contribute to Kubernetes, you'll be collaborating with different
+community members. Having the approval of senior project maintainers means that
+they have seen your work and they believe that you'll be a good addition to the
+project. 
+
+**Motivation:** We want the sponsorships to be meaningful, so we require the sponsors to be
+maintainers that collaborated with you and we also don't want to burden the
+senior maintainers otherwise they'll be reluctant to sponsor anyone. Therefore,
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer.
+
+### 4. Apply for maintainership to contributor leads
+
+The application will be discussed by the lead maintainers and put to vote. If
+the vote result is negative, do not be discouraged! You will be given feedback
+so that you can improve yourself and apply again. We want to empower our
+community and cultivate a trust relationship that's why we carefully evaluate
+abilities and experience of each candidate.
+
+**TBD:** As of now Brian Grant is the wg-contribex lead that approves new maintainers.
+
+## What we expect from our maintainers
+
+First and foremost, be a good person, as cheeky as it sounds, you are now a
```

Comment:
We still need to make it clear what's the difference of this with current implementing `Approvers/Reviewers` process (see https://github.com/kubernetes/kubernetes/pull/36528), especially with Approvers. As @davidopp said, should Approvers always be in  maintainers list (otherwise, how can they approve PR? Ask a maintainer to do?)